api/gdpr.py

```diff
@@ -0,0 +1,165 @@
+from typing import List
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.shortcuts import get_object_or_404
+from helsinki_gdpr.models import SerializableMixin
+from helsinki_gdpr.views import DeletionNotAllowed, DryRunSerializer, GDPRAPIView
+from rest_framework import status
+from rest_framework.response import Response
+
+from tilavarauspalvelu.utils.anonymisation import anonymize_user_data
+
+User = get_user_model()
+
+
+class ProfileUser(SerializableMixin):
+    def __init__(self, user: User, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.user = user
+
+    serialize_fields = (
+        {"name": "user", "accessor": lambda u: u.get_full_name()},
+        {"name": "email"},
+        {"name": "reservations"},
+        {"name": "applications"},
+    )
+
+    @property
+    def reservations(self) -> List[List]:
+        reservations = []
+        for res in self.user.reservation_set.all():
+            reservations.append(
+                [
+                    res.name,
+                    res.description,
+                    res.reservee_first_name,
+                    res.reservee_last_name,
+                    res.reservee_email,
+                    res.reservee_phone,
+                    res.reservee_address_zip,
+                    res.reservee_address_city,
+                    res.reservee_address_street,
+                    res.billing_first_name,
+                    res.billing_last_name,
+                    res.billing_email,
+                    res.billing_phone,
+                    res.billing_address_zip,
+                    res.billing_address_city,
+                    res.billing_address_street,
+                    res.reservee_id,
+                    res.reservee_organisation_name,
+                    res.free_of_charge_reason,
+                    res.cancel_details,
+                ]
+            )
+        return reservations
+
+    @property
+    def applications(self) -> List:
+        applications = []
+        for app in self.user.application_set.all():
+            applications.append(app.additional_information)
+            events = []
+            for e in app.application_events.all():
```

Comment:
Here again, an empty line before the for loop will help readability.

Reply:
So referring to the other answer I partly agree here the spacing. I'll suggest following:
```
applications.append(app.additional_information)

events = []
for e in app.application_events.all():
    events.append(e.name)
```

since the `applications.append....` belongs to the above for loop and the `events = []` here is the sublogical whole with this for loop. I don't know if you can get into my brains with this thinking :brain:

Reply:
Ah, I checked it out locally and this makes total sense! If you add a padding line after `applications.append` it is clear the events variable, the for loop and the append to applications is related.